Cannot validate response with empty content

Hello!
I'm really excited about this project because it perfectly fit my current needs - I want to have a custom open api file, but verify if it's not out of sync with the API.

However, there's one problem I'm having now I need your help.
When I test response of an endpoint that returns no content, I get this error:
```
E           openapi_tester.exceptions.UndocumentedSchemaSectionError: Error: Unsuccessfully tried to index the OpenAPI schema by `content`. 
E           
E           No content documented for method: delete, path: /endpoint/{version}/path
```

I've looked it up in code and it is raised in the `SchemaTester.get_key_value`, which raises error when a key is not defined.
It is however possible to have no `content` key. This way you define an empty response in OpenAPI 3.x.

How could I get it fixed? Should I skip testing empty responses?
I think I could provide a fix for this, but please tell me what is the desired way of solving this problem.

--- a/openapi_tester/schema_tester.py
+++ b/openapi_tester/schema_tester.py
@@ -140,17 +140,28 @@ class SchemaTester:
         if "openapi" not in schema:  # pylint: disable=E1135
             # openapi 2.0, i.e. "swagger" has a different structure than openapi 3.0 status sub-schemas
             return self.get_key_value(status_code_object, "schema")
-        content_object = self.get_key_value(
-            status_code_object,
-            "content",
-            f"\n\nNo content documented for method: {response_method}, path: {parameterized_path}",
-        )
-        json_object = self.get_key_value(
-            content_object,
-            "application/json",
-            f"\n\nNo `application/json` responses documented for method: {response_method}, path: {parameterized_path}",
-        )
-        return self.get_key_value(json_object, "schema")
+
+        if status_code_object.get("content"):
+            content_object = self.get_key_value(
+                status_code_object,
+                "content",
+                f"\n\nNo content documented for method: {response_method}, path: {parameterized_path}",
+            )
+            json_object = self.get_key_value(
+                content_object,
+                "application/json",
+                f"\n\nNo `application/json` responses documented for method: {response_method}, path: {parameterized_path}",
+            )
+            return self.get_key_value(json_object, "schema")
+
+        if response.json():
+            raise UndocumentedSchemaSectionError(
+                UNDOCUMENTED_SCHEMA_SECTION_ERROR.format(
+                    key="content",
+                    error_addon=f"\n\nNo `content` defined for this response: {response_method}, path: {parameterized_path}",
+                )
+            )
+        return {}
 
     def handle_one_of(self, schema_section: dict, data: Any, reference: str, **kwargs: Any):
         matches = 0

--- a/tests/test_schema_tester.py
+++ b/tests/test_schema_tester.py
@@ -191,10 +191,30 @@ def test_validate_response_failure_scenario_undocumented_status_code(monkeypatch
         tester.validate_response(response)
 
 
+def test_validate_response_failure_scenario_undocumented_content(client, monkeypatch):
+    schema = deepcopy(tester.loader.get_schema())
+    del schema["paths"][parameterized_path][method]["responses"][status]["content"]
+    monkeypatch.setattr(tester.loader, "get_schema", mock_schema(schema))
+    response = client.get(de_parameterized_path)
+    with pytest.raises(
+        UndocumentedSchemaSectionError,
+        match=f"Error: Unsuccessfully tried to index the OpenAPI schema by `content`. \n\nNo `content` defined for this response: {method}, path: {parameterized_path}",
+    ):
+        tester.validate_response(response)
+
+
 def test_validate_response_global_case_tester(client):
     response = client.get(de_parameterized_path)
     with pytest.raises(CaseError, match="is not properly PascalCased"):
         SchemaTester(case_tester=is_pascal_case).validate_response(response=response)
+
+
+def test_validate_response_empty_content(client, monkeypatch):
+    schema = deepcopy(tester.loader.get_schema())
+    del schema["paths"][parameterized_path][method]["responses"][status]["content"]
+    monkeypatch.setattr(tester.loader, "get_schema", mock_schema(schema))
+    response = response_factory({}, de_parameterized_path, method, status)
+    tester.validate_response(response)
 
 
 def test_validate_response_global_ignored_case(client):

